File: tests/benchmarks/fib/fib.js
function fib(n) {
    if (n < 2) return n;
    return fib(n - 1) + fib(n - 2);
}

let t0 = performance.now();
let res = fib(35);
let t1 = performance.now();

console.log(res);
console.log("Time: " + ((t1 - t0) / 1000) + "s");

File: tests/benchmarks/fib/fib.py
import time

def fib(n):
    if n < 2:
        return n
    return fib(n - 1) + fib(n - 2)

t0 = time.time()
res = fib(35)
t1 = time.time()

print(res)
print("Python Time: " + str(t1 - t0) + "s")
